pretyping/evarconv.ml

```diff
@@ -1228,6 +1228,7 @@ let solve_unconstrained_impossible_cases env evd =
   Evd.fold_undefined (fun evk ev_info evd' ->
     match ev_info.evar_source with
     | loc,Evar_kinds.ImpossibleCase ->
+       (* Why does this ad-hoc code have to be here instead of cases.ml? *)
```

Comment:
Sorry I did not mean to include this comment in this PR. Thanks for the answer anyway. It looks to me as if this example could trigger an error (because an inaccessible branch is provided), so I wonder if it is only for compatibility. In OCaml 4.03, there is a warning in similar situations, not an error though.